terraform-plan/README.md

```diff
@@ -243,6 +243,24 @@ The [dflook/terraform-apply](https://github.com/dflook/terraform-github-actions/
   - Optional
   - Default: 10
 
+* `TERRAFORM_PRE_RUN`
```

Comment:
The doc change in this file is good - the other README.md files have the example in the wrong place and type information missing.

Reply:
I've updated all the documentation to be consistend and extended the workflow test also to apply.